radicle-httpd/src/api/test.rs

```diff
@@ -0,0 +1,133 @@
+use std::path::Path;
+use std::sync::Arc;
+use std::{env, fs};
+
+use axum::body::Body;
+use axum::http::Request;
+use axum::response::Response;
+use axum::Router;
+use rand::distributions::{Alphanumeric, DistString};
+use serde_json::Value;
+use tower::ServiceExt;
+
+use radicle::cob::issue::Issues;
+use radicle::git::raw as git2;
+use radicle::storage::WriteStorage;
+use radicle_cli::commands::rad_init;
+
+use crate::api::Context;
+
+pub const HEAD: &str = "1e978d19f251cd9821d9d9a76d1bd436bf0690d5";
+pub const HEAD_1: &str = "f604ce9fd5b7cc77b7609beda45ea8760bee78f7";
+
+pub fn seed() -> Context {
+    let random_suffix = Alphanumeric.sample_string(&mut rand::thread_rng(), 16);
+    let temp_dir = env::temp_dir()
```

Comment:
We prefer `tempfile::tempdir()`, because it gets auto-cleaned up at the end of the tests. Otherwise we're accumulating a bunch of data very quickly. However you'll have to pass it as a reference into this function for it to be dropped at the end of the test, and not the end of this function.

Reply:
It's already a dependency though, and avoids needing `rand` which you pulled in.

Reply:
[fixed](https://github.com/radicle-dev/heartwood/compare/0a4dd930144842337392c0d92496535c5e44d38a..a8270fefa9ef77337c08fd7f84b9d0db9428c23d), created wrapper `Response` too